docs/docs/how-to/disable-api-database.md

```diff
@@ -18,6 +18,16 @@ rm -rf api
 
 You can also run `yarn install` to cleanup those packages that aren't used any more.
 
+## Disable Prisma functionality
+The `--prisma` and `--dm` flags are set to `true` by default and need to be set to `false` in the build command.
+
+```toml {4}
+[build]
```

Comment:
@jtoar Can we use the `--prisma` and `--dm` flags on "any" deployment?

That way perhaps this document can read a little less Netlify-centric.

Reply:
@dthyresson those flags are valid for netlify, vercel, and layer0. The deploy commands for all those providers use the `deployBuilder` from helpers:
- https://github.com/redwoodjs/redwood/blob/70b42c7193e1db68b4e44054fd94ed9f81b4d809/packages/cli/src/commands/deploy/netlify.js#L1
- https://github.com/redwoodjs/redwood/blob/70b42c7193e1db68b4e44054fd94ed9f81b4d809/packages/cli/src/commands/deploy/vercel.js#L1
- https://github.com/redwoodjs/redwood/blob/70b42c7193e1db68b4e44054fd94ed9f81b4d809/packages/cli/src/commands/deploy/layer0.js#L12

Reply:
Render and Flightcontrol actually takes a similar set of flags so it may be possible to only deploy the web side there too:

- flightcontrol https://github.com/redwoodjs/redwood/blob/70b42c7193e1db68b4e44054fd94ed9f81b4d809/packages/cli/src/commands/deploy/flightcontrol.js#L17-L37
- render https://github.com/redwoodjs/redwood/blob/70b42c7193e1db68b4e44054fd94ed9f81b4d809/packages/cli/src/commands/deploy/render.js#L15-L30

Reply:
Lastly, serverless takes a "side". But I don't see any logic around prisma or data-migrate. There may still need to be some work done here: https://github.com/redwoodjs/redwood/blob/main/packages/cli/src/commands/deploy/serverless.js.